test/fixtures/rules/std_rule_cases/L010.yml

```
@@ -62,3 +62,13 @@ test_fail_date_part_capitalisation_policy_upper:
 test_pass_date_part_consistent_capitalisation:
   # Test that correctly capitalized time units are left unchanged
   pass_str: SELECT dt + INTERVAL 2 DAY, INTERVAL 3 HOUR
+
+test_pass_capitalisation_policy_data_type_keywords:
+  # Test cases where data types are keywords but are handled by L059 instead of L010
+  # See: https://github.com/sqlfluff/sqlfluff/pull/2121
+  fail_str: create table table1 (account_id bigint, column_two serial);
+  fix_str: CREATE TABLE table1 (account_id bigint, column_two serial);
```

```suggestion
  pass_str: CREATE TABLE table1 (account_id bigint, column_two serial);
```

While both versions of this test show that L010 is not affecting serial, it is much clearer to ask the question "has anything changed?" vs "are some of the things that have changed not correct?"